README.md

```diff
@@ -32,6 +32,7 @@ section like the following:
   # php path. run 'which php' to find the path
   'executablePath': /usr/bin/php
 ```
+> Additional settings can be found under the supporting package **Linter** automatically installed with linter-php. Global settings like linting delay can be adjusted here.
```

Comment:
How about something like this?

```markdown
_Additional settings can be found in the consumer of your linter providers, typically this is the [Linter](https://atom.io/packages/linter) package automatically installed with `linter-php`. Global settings like linting delay must be adjusted here._
```

This changes the wording to accommodate the fact that Linter isn't the _only_ consumer of linter-providers (Nuclide can use them natively). I also updated the style to a note instead of a quote.

Feel free to further tweak the wording as I'm sure there is a better way to say that 😛.